### PR TITLE
Created a new section Deploy Managed Installer.

### PR DESCRIPTION
**Pull request by v-kikl**
@jgeurten, let me know if this is how you imagined the Deploy Managed Installer section look like.
@jsuther1974.

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 1]
```diff
@@ -214,3 +273,27 @@ Ea Value Length: 7e
 ## Enabling managed installer logging events
 
 Refer to [Understanding Application Control Events](event-id-explanations.md#optional-intelligent-security-graph-isg-or-managed-installer-mi-diagnostic-events) for information on enabling optional managed installer diagnostic events.
+
+## Deploying the Managed Installer
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:277]: Let's call this "Deploying the Managed Installer rule collection"

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 2]
```diff
@@ -214,3 +273,27 @@ Ea Value Length: 7e
...
+
+Once you've completed configuring your chosen Managed Installer, by specifying which to use in the AppLocker policy, enabling the service enforcement of it, and by enabling the Managed Installer option in a WDAC policy, you'll need to deploy it.
+
+1. Using the following command to deploy the policy.
+   ```powershell
+   Set-AppLockerPolicy -XmlPolicy $policyFile -Merge -ErrorAction SilentlyContinue
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:283]: $policyFile= 
@"
      Raw_AppLocker_Policy_XML
"@

Set-AppLockerPolicy -XmlPolicy $policyFile -Merge -ErrorAction SilentlyContinue

**Review by jgeurten — CHANGES_REQUESTED** [review]

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 3]
```diff
@@ -214,3 +273,27 @@ Ea Value Length: 7e
...
+   ```
+
+2. Verify policy deployment
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:286]:
```suggestion
2. Verify Deployment of the Rule set was successful
```

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 4]
```diff
@@ -214,3 +273,27 @@ Ea Value Length: 7e
...
+   ```powershell
+   Get-AppLockerPolicy -Local
+   
+   Version RuleCollections RuleCollectionTypes
+   ------- --------------- -------------------
+      1 {0, 0, 0, 0...} {Appx, Dll, Exe, ManagedInstaller...}
+   ```
+    Notice the output shows the ManagedInstaller rule is there.
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:294]:
```suggestion
    Verify the output shows the ManagedInstaller rule set.
```

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 5]
```diff
@@ -214,3 +273,27 @@ Ea Value Length: 7e
...
+
+3. Get the policy XML (optional) using PS:
+   ```powershell
+   Get-AppLockerPolicy -Effective -Xml -ErrorVariable ev -ErrorAction SilentlyContinue
+   ```
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:299]: At the end, mention that this command will show the raw XML to verify the individual rules (re-word this section)

**Review by jgeurten — CHANGES_REQUESTED** [review]

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 6]
```diff
@@ -93,27 +93,86 @@ Currently, neither the AppLocker policy creation UI in GPO Editor nor the PowerS
     <RuleCollection Type="ManagedInstaller" EnforcementMode="AuditOnly">
     ```
 
-An example of a valid Managed Installer rule collection using Microsoft Endpoint Config Manager (MEMCM) is shown below.
+An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), Powershell, and Powershell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:96]:
```suggestion
An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), PowerShell, and PowerShell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.
```

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 7]
```diff
@@ -214,3 +273,32 @@ Ea Value Length: 7e
 ## Enabling managed installer logging events
 
 Refer to [Understanding Application Control Events](event-id-explanations.md#optional-intelligent-security-graph-isg-or-managed-installer-mi-diagnostic-events) for information on enabling optional managed installer diagnostic events.
+
+## Deploying the Managed Installer
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:277]:
```suggestion
## Deploying the Managed Installer rule collection
```

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 8]
```diff
@@ -214,3 +273,32 @@ Ea Value Length: 7e
...
+
+Once you've completed configuring your chosen Managed Installer, by specifying which to use in the AppLocker policy, enabling the service enforcement of it, and by enabling the Managed Installer option in a WDAC policy, you'll need to deploy it.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:279]:
```suggestion
Once you've completed configuring your chosen Managed Installer by specifying which option to use in the AppLocker policy, enabling the service enforcement of it, and enabling the Managed Installer option in a WDAC policy, you'll need to deploy it.
```

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 9]
```diff
@@ -214,3 +273,32 @@ Ea Value Length: 7e
...
+
+1. Using the following command to deploy the policy.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:281]:
```suggestion
1. Use the following command to deploy the policy.
```

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 10]
```diff
@@ -214,3 +273,32 @@ Ea Value Length: 7e
...
+   ```powershell
+   $policyFile=
+   @"
+   Raw_AppLocker_Policy_XML
+   "@
+   Set-AppLockerPolicy -XmlPolicy $policyFile -Merge -ErrorAction SilentlyContinue
+   ```
+
+2. Verify Deployment of the Rule set was successful
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:290]:
```suggestion
2. Verify deployment of the ruleset was successful.
```

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 11]
```diff
@@ -214,3 +273,32 @@ Ea Value Length: 7e
...
+   ```powershell
+   Get-AppLockerPolicy -Local
+   
+   Version RuleCollections RuleCollectionTypes
+   ------- --------------- -------------------
+   1 {0, 0, 0, 0...} {Appx, Dll, Exe, ManagedInstaller...}
+   ```
+   Verify the output shows the ManagedInstaller rule set.
+
+3. Get the policy XML (optional) using PS:
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:300]:
```suggestion
3. Get the policy XML (optional) using PowerShell:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-kikl, kindly commit the suggested changes if you agree with them. Many thanks.

File: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md [hunk 12]
```diff
@@ -93,27 +93,86 @@ Currently, neither the AppLocker policy creation UI in GPO Editor nor the PowerS
     <RuleCollection Type="ManagedInstaller" EnforcementMode="AuditOnly">
     ```
 
-An example of a valid Managed Installer rule collection using Microsoft Endpoint Config Manager (MEMCM) is shown below.
+An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), Powershell, and PowerShell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md:96]:
```suggestion
An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), PowerShell, and PowerShell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**